presentations/cargo/slides.adoc

```diff
@@ -62,14 +62,16 @@ The debug profile is default here.
 
 Test use the debug profile per default.
 
-`cargo test` runs all test, including documentation test and compilation of examples!
+`cargo test --all --examples` runs all test, including documentation test and compilation of examples!
 
 == `cargo bench`
 
 `cargo bench` runs all benchmarks.
 
 Benchmarks default to the release profile.
 
+Benchmarks only work on the nightly compiler.
```

Comment:
Is it worth mentioning `criterion` here for stable benches?

Reply:
We probably should.

Reply:
```suggestion
Benchmarks can be written using the `criterion` crate.
```